pkg/collector/corechecks/oracle-dbm/connection_handling.go

```diff
@@ -57,7 +57,7 @@ func (c *Check) Connect() (*sqlx.DB, error) {
 	}
 	c.driver = oracleDriver
 
-	log.Infof("driver: %s", oracleDriver)
+	log.Infof("%s driver: %s", c.logPrompt, oracleDriver)
```

Comment:
Is there a case where `c`.logPrompt` won't have been set yet when we get to this line?